.github/workflows/action-semantic-pull-request.yml

```
@@ -0,0 +1,27 @@
+name: "Semantic PR"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
```

The general policy appears to be to depend on specific commits of third-party actions rather than version numbers/ranges.

---

Good catch! Yeah, let's change it to
```suggestion
      - uses: amannn/action-semantic-pull-request@01d5fd8a8ebb9aafe902c40c53f0f4744f7381eb # v5.0.2
```